seleniumByTechnocredits/src/seleniumPractise/saket/Assignment_1.java

```diff
@@ -0,0 +1,62 @@
+package seleniumPractise.saket;
+
+import org.omg.Messaging.SyncScopeHelper;
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import util.Init;
+
+public class Assignment_1 {
+
+	public static void main(String[] args) throws InterruptedException {
+
+		WebDriver driver = Init.initChromeDriver();
+		driver.get("http://automationbykrishna.com/#");
+
+		driver.findElement(By.id("basicelements")).click();
+		Thread.sleep(2000);
+
+		driver.findElement(By.id("UserFirstName")).sendKeys("Saket");
+
+		driver.findElement(By.xpath("//input[@id='UserLastName']")).sendKeys("Nagmote");
+
+		driver.findElement(By.xpath("//input[@id='UserCompanyName']")).sendKeys("Capgemini");
+
+		String b = driver.findElement(By.id("UserFirstName")).getAttribute("value");
+		String c = driver.findElement(By.xpath("//input[@id='UserLastName']")).getAttribute("value");
+		String d = driver.findElement(By.xpath("//input[@id='UserCompanyName']")).getAttribute("value");
+
+		driver.findElement(By.xpath("//button[@onclick='myFunctionPopUp()' ]")).click();
+
+		Thread.sleep(2000);
+
+		Alert alert = driver.switchTo().alert();
+
+		String a = alert.getText();
+		System.out.println(a);
+
+		String[] words = a.split("and");
```

Comment:
These many lines of code can be optimised as 

	String[] words = a.split("and");

	 		String input = (b.concat(c).concat(d));
			System.out.println("Input" + input);
			String expected = "";

	 		for (int i = 0; i <= words.length - 1; i++) {

	 			expected = expected + words[i];

	 		}
			System.out.print(expected);

	 		a = expected.replaceAll("\\s", "");
 ==================================
optimised code
	String expectedtext = s1 + " and " + s2 + " and " + s3;
String actualtext = alert.getText();
		if (actualtext.equals(expectedtext)) {

Reply:
agreed and completed the changes in the code!